lib/datadog/appsec/contrib/rails/auto_instrument_railtie.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Datadog
+  module AppSec
+    module Contrib
+      module Rails
+        # Railtie to include AutoInstrumentation in rails loading
+        class AutoInstrumentRailtie < ::Rails::Railtie
```

Comment:
Do we need a require of railtie here to guarantee its presence?

Reply:
it's a bit finicky - we check if railtie is loaded before requiring this file:
https://github.com/DataDog/dd-trace-rb/pull/4435/files#diff-6932a928693e0818dc0bb6586852b0453f1020945ae10f7ac655161f24c1a979R11-R12